src/lib/loaders/legacy/total.js

```diff
@@ -23,6 +23,26 @@ export const total = (path, accessToken, options = {}) => {
   }))
 }
 
+export const totalViaLoader = (loader, loaderOptions, apiOptions = {}) => {
```

Comment:
This accepts a loader, and returns a promise that resolves to the total count.

Reply:
This is in a ‘legacy’ file, should it move to a new file?

Reply:
Yes! I was grepping for `/legacy` to track down all the remaining pieces and realized this entire file is in there (whereas `all` is just in `lib/`).

So yes, let's leave for now, but as I try to finish up everything for these this sprint, I will move it at some point.